pkgs/development/python-modules/cardimpose/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pymupdf,
+}:
+buildPythonPackage rec {
+  pname = "cardimpose";
+  version = "0.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7GyLTUzWd9cZ8/k+0FfzKW3H2rKZ3NHqkZkNmiQ+Tec=";
+  };
+
+  build-system = [ setuptools ];
```

```suggestion
  nativeBuildSystem = [
    pypaInstallHook
    setuptoolsBuildHook
  ];
```
If you set `pyproject = false`, you'll need to replace this. Otherwise it's fine